CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+## [2.0.0] - 2020-11-03
```

Comment:
Change this to

## Unreleased
### Changed
* Updated to follow the standard Koop Framework. The wrapper project will be maintained elsewhere.